app/scripts/views/behaviors/settings-after-verification.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Auth broker to handle users who browse directly to the site.
```

Comment:
This is a copy/paste comment and needs to be updated. The module also needs a test.